command/src/command.proto

```diff
@@ -760,6 +760,8 @@ message ProtobufAccessLog {
     required string tag = 18;
     // POSIX timestamp, nanoseconds
     required Uint128 time = 19;
+    // entire time between first byte received and last byte of the response
+    required uint64 request_time = 20;
```

Comment:
Should be optional as well, besides what happens if there is a trouble somewhere in the pipeline

Reply:
An access log is produced at the very moment a pipeline (rather called "session") is over, be it successfully (`log_request_success`) or unsuccessfully (`log_request_error`).
The `request_time` metric is produced by a `SessionMetrics` object that computes time elapsed from the beginning of the session.
Do you think this behaviour is okay? Should we document it better?

We could add an additionaal line to this comment:

```
If a request ends abruptly before the last byte is transmitted,
the `request_time` produced is the time elapsed from the first byte received.
```

Reply:
done!